Clustering/Hdbscan/src/test/java/org/tribuo/clustering/hdbscan/TestHdbscan.java

```diff
@@ -19,12 +19,7 @@
 import com.oracle.labs.mlrg.olcut.util.Pair;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
-import org.tribuo.DataSource;
-import org.tribuo.Dataset;
-import org.tribuo.Feature;
-import org.tribuo.Model;
-import org.tribuo.MutableDataset;
-import org.tribuo.Prediction;
+import org.tribuo.*;
```

Comment:
Could you flip this back to an explicit import here and for `java.util.*`?

Reply:
Yes, this was supposed to come in with the previous commit. The comment mentioned it but that doesn't really help.